### PR TITLE
feat: sync accepted marketing to cio

### PR DESCRIPTION
**Pull request by idoshamun**
(Un)subscribe users from the relevant topic based on the `acceptedMarketing` property

### CODE REVIEW

File: src/cio.ts [hunk 1]
```diff
@@ -47,5 +47,10 @@ export async function identifyUser(
     created_at: dateToCioTimestamp(debeziumTimeToDate(dup.createdAt)),
     updated_at: dateToCioTimestamp(debeziumTimeToDate(dup.updatedAt)),
     referral_link: genericInviteURL,
+    cio_subscription_preferences: {
+      topics: {
+        topic_4: user.acceptedMarketing,
```
Comment (omBratteng) [inline src/cio.ts:52]: What is `topic_4`?

File: __tests__/workers/userUpdatedCio.ts [hunk 2]
```diff
@@ -59,6 +60,12 @@ describe('userUpdatedCio', () => {
       updated_at: 1714577744,
       username: 'cio',
       referral_link: referral,
+      accepted_marketing: true,
+      cio_subscription_preferences: {
+        topics: {
+          topic_4: true,
+        },
+      },
```
Comment (rebelchris) [inline __tests__/workers/userUpdatedCio.ts:68]: Maybe we should add a test for false as well, just to verify down the line if breaking stuff comes?

**Review by rebelchris — APPROVED** [review]
Suggestion, but non blocking

**Review by omBratteng — APPROVED** [review]

File: __tests__/workers/userUpdatedCio.ts (hunk 2 above)
Comment (idoshamun) [reply __tests__/workers/userUpdatedCio.ts:68]: Done

**Pull request merged**